src/components/Modal/AnalyticsModal/index.tsx

```diff
@@ -1,81 +0,0 @@
-import { FC } from "react"
-import {
-  BodyLg,
-  BodyXs,
-  Button,
-  H5,
-  Image,
-  ModalBody,
-  ModalFooter,
-  useColorModeValue,
-  VStack,
-  ModalCloseButton,
-  ModalHeader,
-  Divider,
-} from "@threshold-network/components"
-import { BaseModalProps } from "../../../types"
-import withBaseModal from "../withBaseModal"
-import { useModal } from "../../../hooks/useModal"
-import { ModalType } from "../../../enums"
-import { FeedbackSubmissionType } from "../FeedbackSubmissionModal"
-import FeedbackAnalyticsInfo from "../../FeedbackAnalyticsInfo"
-import analyticsImageLight from "../../../static/images/AnalyticsIllustration.png"
-import analyticsImageDark from "../../../static/images/AnalyticsIllustrationDark.png"
```

Comment:
Let's remove those two images from the repo.

Reply:
Ref commit: 080cff06